libraries/lib-audio-io/AudioIO.cpp

```diff
@@ -148,6 +148,7 @@ struct AudioIoCallback::TransportState {
                assert(false);
                continue;
             }
+            //TODO: initialize with sequence own channels count and sampleRate
```

Comment:
The comment isn't specifically about master channel. 
When we allocate two mono effects to one stereo track there is at least one problem with UI : you can't see the output of the second channel. 
I think configuring effects to process samples in native track format and rate could have benefit of reducing CPU load in case if playback sample rate is higher compared to playback rate. But that may be conceptually wrong.

Reply:
Regarding the CPU, I don't know, it looks like there is no negotiation at the moment: the framework dictates the sampling rate to the effect instances, which I guess is much simpler. Also, I think resampling is inevitable, so I'm not sure we want to try and minimize it.
My point is that this TODO should probably be tempered with some doubt, e.g., "consider initializing (...) because (...)"